esp/services/esdl_svc_engine/esdl_store.cpp

```diff
@@ -63,7 +64,7 @@ class CEsdlSDSStore : implements IEsdlStore, public CInterface
         if (!conn)
            throw MakeStringException(-1, "Unable to connect to ESDL Service definition information in dali '%s'", xpath.str());
 
-        conn->close(false); //release lock right away
+        conn->close(); //release lock right away
```

Comment:
Interacting with the connection (e.g. conn->queryRoot()) after it's closed is not safe.
I'm surprised the toXML doesn't crash actually.

Reply:
I'm also surprised that it actually worked. I guess there might've been some confusion about what the default "false" parameter implies. I'll move the calls to the end of the operations.

Reply:
It would be worth adding a comment to clarify on close.
close(true) means delete the node it was connected to.